### PR TITLE
chore: remove unused `/charter` page

### PR DESCRIPTION
**Pull request by CommanderStorm**
This PR removes this deprecation warning:
image

The deprecation comes from this page:
image

This PR does remove said page as it is not used by https://maplibre.org/about/ any longer.

### CODE REVIEW

File: layouts/shortcodes/charter.html [hunk 1]
```diff
@@ -1,8 +1,13 @@
-{{ $headers := dict }}
+{{ $url := "https://api.github.com/repos/maplibre/maplibre/contents/CHARTER.md?ref=main" }}
+{{ $cacheKey := print $url (now.Format "2006-01-02") }}
```
Comment (CommanderStorm) [inline layouts/shortcodes/charter.html:2]: why I added this: https://gohugo.io/functions/resources/getremote/#caching

Given that the site is build in CI might not be a real issue, but (if the charter gets changed in the future) might prevent dev-confusion why this is not updated..

**Comment by HarelM** [comment]
I'll be happy to approve this, but I'm not sure I fully understand everything I read...
Cc: @ramyaragupathy @louwers

**Comment by louwers** [comment]
Honestly this whole approach seems overcomplicated to me.

**Review by louwers — CHANGES_REQUESTED** [review]
This shortcode and the /charter page should be removed.

We link directly to the version on GitHub now. https://maplibre.org/about/

**Review by louwers — APPROVED** [review]

**Comment by louwers** [comment]
Thank you. 🙏

**Pull request merged**